code/modules/research/xenoarchaeology/artifact/artifact_xenomarker.dm

```diff
@@ -0,0 +1,59 @@
+#define NECROMARKER_MIN_WHISPER_INTERVAL 200
+#define NECROMARKER_MAX_WHISPER_INTERVAL 450
+/obj/machinery/necromarker
+    name = "mysterious sculpture"
+    desc = "A mysterious scultpure of spiral spines, covered in runes."
+    icon = 'icons/xenoarch_icons/necromarker.dmi'
+    icon_state = "black"
+    density = 1
+
+    var/ticks_not_whispered = 0
+    var/next_whisper = 300
+    var/whispers = list("...bring me flesh...", "...make us whole...", "...we must be whole...", "...join us in unity...", "...one mind, one soul, one flesh...", "...MAKE US WHOLE...")
+
+    machine_flags = WRENCHMOVE
+
+/obj/machinery/necromarker/New()
+    . = ..()
+
+/obj/machinery/necromarker/MouseDrop_T(mob/M as mob, mob/user as mob)
+    if(!istype(M))
+        return
+    if(Adjacent(user))
+        Consume(M)
+
+/obj/machinery/necromarker/Consume(mob/M as mob, mob/user as mob)
+    if(anchored && ismob(M) && Adjacent(M))
+        M.forceMove(src.loc)
+        var/mob/living/simple_animal/hostile/necro/zombie/Z = new(src.loc)
```

Comment:
This aint a necromorph, this is a zombie

/mob/living/simple_animal/hostile/monster/necromorph is the big scary necromorph